fetch-in-react/my-app/src/Users.tsx

```diff
@@ -0,0 +1,82 @@
+/* eslint-disable @typescript-eslint/no-unused-vars -- Remove me */
+import { useEffect, useState } from 'react';
+import { UserDetails } from './UserDetails';
+import { UserCard } from './UserCard';
+import './Users.css';
+
+export type User = {
+  id: number;
+  name: string;
+  email: string;
+  company: {
+    name: string;
+  };
+};
+export function Users() {
+  const [isLoading, setIsLoading] = useState(true);
+  const [error, setError] = useState<unknown>();
+  const [users, setUsers] = useState<User[]>([]);
+  const [user, setUser] = useState<User>();
+
+  useEffect(() => {
+    async function getUsers() {
+      try {
+        const resp = await fetch('https://jsonplaceholder.typicode.com/users');
+        const usersJSON = await resp.json();
+        const tenUsers: User[] = [];
+        for (let i = 0; i < 10; i++) {
```

Comment:
I see you took the "list of ten users" seriously! Actually, the API you are calling will only return 10. I hadn't anticipated that someone would actually truncate the response to the first 10!